### ISSUE TITLE
Loaded Payments Module - Error Handling Broken

### ISSUE THREAD

**Issue by tekyguy** [loadedcommerce/loaded7#641]
If there is any kind of decline, no error message is shown and the cart goes back to a any empty payment form.

**Comment by tekyguy** [comment]
Is it done though?  We need more friendly errors that customers can understand.  Should we make a new bug?

**Comment by saliozzia** [comment]
yes, that would be a new issue, this was just error handling and getting
display to work

the new issue would need to define the content of the error message.
typically we show the message the gateway responds with.


Best Regards,

Salvatore Iozzia

Founder and CEO
Loaded Commerce



loadedcommerce.com <http://www.loadedcommerce.com/> | loaded7.com
<http://www.loaded7.com/>

Cell: 0031 6 52562533
Skype: chainreactionweb
twitter: @saliozzia @loadedcommerce
